tests/unit/package_managers/conftest.py

```diff
@@ -6,6 +6,7 @@
 import git
 import pytest
 
+from cachi2.core.models.input import Request
```

Comment:
We won't have this file after #657, so you need to rebase and move it to the global conftest.